docs/product-assembly_designing-and-building-cases/src/main/asciidoc/case-management-dynamic-user-task-API-proc.adoc

```diff
@@ -0,0 +1,60 @@
+[id='case-management-dynamic-user-task-API-proc']
+= Creating a dynamic user task using the REST API
+
+You can create a dynamic user task during case run time using the REST API. To create a dynamic user task, the following information must be given:
+
+* task name
+* task description (optional, but recommended)
+* actors
+* groups
```

Comment:
either actor or groups or both can be given, it's not required to provide both users and groups